template_feed/Microsoft.DotNet.Common.ItemTemplates/content/PackagesProps/Directory.Packages.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+<!--#if inherit -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Packages.props', '$(MSBuildThisFileDirectory)../'))"
```

Comment:
I think the root `Directory.Packages.props` should not try to inherit from any parents.  Would it be possible to add an option to the template so by default it creates the `Directory.Packages.props` with just the PropertyGroup and then with some command-line argument it makes a `Directory.Packages.props` with this inheritance?

Reply:
@jeffkl that's all already wired up with the template symbol `inherit` defined in the [template.json](https://github.com/dotnet/sdk/pull/36387/files#diff-b5221c4a707625fac869d1c5ba3fa00b20f62de0e6657ffdaf4497bdcd157434R21-R26) - currently the default invocation of this template (much like the Directory.Build.props/targets templates we also have in the SDK) implicitly passes `inherit=false`, but a user can pass `--inherit true` on the CLI and then the Import will be added.

Reply:
Perfect, thanks for clarifying.